### PR TITLE
Remove PII from log

### PR DESCRIPTION
**Pull request by ghost**
In case of error, the debug log would output PII.

### CODE REVIEW

**Review by KillianG — APPROVED** [review]

File: lib/auth/Vault.ts [hunk 1]
```diff
@@ -26,7 +33,20 @@ function vaultSignatureCb(
...
         return callback(err);
     }
-    log.debug('received info from Vault', { authInfo });
+
+    const { email, ...userInfoWithoutEmail } = authInfo.message.body.userInfo;
+
+    log.debug('received info from Vault', {
+        ...authInfo,
+        message: {
+            message: authInfo.message,
```
Comment (welansari) [inline lib/auth/Vault.ts]:
```suggestion
            message: authInfo.message.message,
```

File: lib/auth/Vault.ts [hunk 2]
```diff
@@ -12,7 +12,14 @@ import AuthInfo from './AuthInfo';
  */
 function vaultSignatureCb(
     err: Error | null,
-    authInfo: { message: { body: any } },
+    authInfo: {
+        message: {
+            message: string, body: {
+                userInfo: AuthInfoType,
```
Comment (welansari) [inline lib/auth/Vault.ts]:
```suggestion
        message: {
            message: string,
            body: {
                userInfo: AuthInfoType,
```

Comment (ghost) [reply lib/auth/Vault.ts]: Done + fixed the type for the quotas

**Review by welansari — APPROVED** [review]

**Pull request merged**